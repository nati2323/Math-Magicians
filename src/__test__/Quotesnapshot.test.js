import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import  Quote from '../pages/Quote';

it('Home renders correctly', () => {
  const tree = renderer.create(
      <Quote />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});