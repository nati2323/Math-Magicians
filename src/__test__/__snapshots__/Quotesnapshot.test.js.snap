// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<div
  className="container"
>
  <h3>
    Mathematics is not about numbers, equations, computations, or algothrims: it is about understanding. - William Paul Thurston
  </h3>
</div>
`;
